readme.md

```diff
@@ -8,14 +8,12 @@ You might want to check out [XO](https://github.com/xojs/xo), which includes thi
 
 [**Propose or contribute a new rule ➡**](.github/contributing.md)
 
-
 ## Install
 
-```
+```bash
```

Comment:
restore this,since there is `$` inside

Reply:
there is `$` inside

Reply:
wouldn't it be better to have it conforming to markdownlint? 

https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md040

Reply:
```suggestion
```console
```

works better